### PR TITLE
Add page for prefers-reduced-transparency

### PR DESCRIPTION
**Pull request by CanadaHonk**
### Description

Added missing page for the `prefers-reduced-transparency` media query (similar to `prefers-reduced-motion`, etc).

### Motivation

The page is missing/does not exist, but exists for all other media queries in CSS Media Queries Level 5.

### Additional details

Spec: https://www.w3.org/TR/mediaqueries-5/#descdef-media-prefers-reduced-transparency

### CODE REVIEW

**Comment by teoli2003** [comment]
Thank you! Does any browser implement this yet?

**Comment by CanadaHonk** [comment]
Not yet, but `prefers-reduced-data` exists from the same spec without any implementation. BCD already exists for this FYI. Also interested in getting it in Firefox hopefully soon.

**Comment by CanadaHonk** [comment]
Oh, looks like the BCD data was added but later removed?
Added in: https://github.com/mdn/browser-compat-data/pull/5720
Removed in: https://github.com/mdn/browser-compat-data/pull/6854 (?)

**Comment by teoli2003** [comment]
Our policy is to wait at least for it to be implemented behind a flag (i.e., until it has a bcd entry). [There have been exceptions that were erroneous decisions, most of them dating from the time we were a wiki]

To save this valuable piece of work, we should mark this PR "on-hold" (a GitHub label we set) and reactivate it as soon as it lands somewhere.

We do this for several other features. Hopefully, this won't be too long.

**Comment by CanadaHonk** [comment]
Sure, makes sense, thanks.

**Comment by hamishwillee** [comment]
When the time comes, usually there is a corresponding HTTP client hint. If so, worth cross linking. For example: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Sec-CH-Prefers-Reduced-Motion

**Comment by CanadaHonk** [comment]
Thanks for the heads up! One does indeed exist: https://wicg.github.io/user-preference-media-features-headers/#sec-ch-prefers-reduced-transparency

Also, the patch I wrote for adding `prefers-reduced-transparency` to Firefox is approved so should hopefully be in Nightly in the next few days behind a pref 🎉

**Comment by hamishwillee** [comment]
@CanadaHonk Nice. Feel free to ping us when that goes in so we can review this ... all the fun work of adding back into BCD has to happen too.

**Comment by teoli2003** [comment]
We discussed yesterday in our editorial meeting how to deal with PRs we want to keep until a feature lands, like this one. We decided to mark them as Draft (in addition to the `on hold` label). The idea is that reviewers scanning the open PRs doesn't hit this PR each time they scan the list. I will convert it (it doesn't change anything practically, it is only bookkeeping for us)

**Comment by CanadaHonk** [comment]
Firefox 113 now supports this behind a pref (`layout.css.prefers-reduced-transparency.enabled`) - created a BCD PR for it - https://github.com/mdn/browser-compat-data/pull/19135

**Comment by teoli2003** [comment]
As soon as it lands (on BCD), we'll be able to reactivate this PR :-)

**Comment by CanadaHonk** [comment]
Landed in BCD so this should be good to review now :)

File: files/en-us/web/css/@media/prefers-reduced-transparency/index.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+- `reduce`
+  - : Indicates that user has notified the system that they prefer an interface that minimizes the amount of transparent or translucent layer effects.
+
+## User preferences
+
+Currently no user agent implements this feature, although various operating systems do support such preferences and if this media query is ever implemented user agents will likely rely on the settings provided by the operating system.
+
+## Examples
+
+> **Note:** No browser currently implements this feature so the following example will not work.
```
Comment (teoli2003) [inline files/en-us/web/css/@media/prefers-reduced-transparency/index.md:29]: Neither this

File: files/en-us/web/css/@media/prefers-reduced-transparency/index.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+slug: Web/CSS/@media/prefers-reduced-transparency
+page-type: css-media-feature
+status:
+  - experimental
+browser-compat: css.at-rules.media.prefers-reduced-transparency
+---
+
+{{CSSRef}}{{SeeCompatTable}}
+
+> **Note:** This feature is not supported by any user agent and its specifics are subject to change.
```
[the remark this answers is not in the document]
Comment (CanadaHonk) [reply files/en-us/web/css/@media/prefers-reduced-transparency/index.md:12]: It's behind an off by default pref in 113 so I think it's best to keep?

**Comment by dipikabh** [comment]
113 will be out only in May (https://wiki.mozilla.org/Release_Management/Calendar#Future_branch_dates). I think we should still keep this on hold and as Draft.

I have added a doc issue for this (as we do for tracking our other doc requirements from bugzilla).

**Comment by CanadaHonk** [comment]
Sure.

**Review by dipikabh — CHANGES_REQUESTED** [review]
Hello @CanadaHonk, we can now move this PR out of draft for Fx113.

I've suggested some edits. (I've made some similar language edits to the `prefers-reduced-motion` feature in https://github.com/mdn/content/pull/26630).

Thanks a lot for adding this page! 🙌

File: files/en-us/web/css/@media/prefers-reduced-transparency/index.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+{{Specifications}}
+
+## Browser compatibility
+
+{{Compat}}
+
+## See also
+
+- CSS [prefers-reduced-motion](/en-US/docs/Web/CSS/@media/prefers-reduced-motion) media query
```
Comment (dipikabh) [inline files/en-us/web/css/@media/prefers-reduced-transparency/index.md:63]: - [Using media queries](/en-US/docs/Web/CSS/Media_Queries/Using_media_queries)

Comment (dipikabh) [inline files/en-us/web/css/@media/prefers-reduced-transparency/index.md:63]: We can also add another link here:

```suggestion
- CSS [prefers-reduced-motion](/en-US/docs/Web/CSS/@media/prefers-reduced-motion) media query
- [Using media queries](/en-US/docs/Web/CSS/Media_Queries/Using_media_queries)
```

Comment (dipikabh) [inline files/en-us/web/css/@media/prefers-reduced-transparency/index.md:63]: We can also add this link here:

```suggestion
- CSS [prefers-reduced-motion](/en-US/docs/Web/CSS/@media/prefers-reduced-motion) media query
- [Using media queries](/en-US/docs/Web/CSS/Media_Queries/Using_media_queries)
```

**Review by dipikabh — CHANGES_REQUESTED** [review]

**Comment by CanadaHonk** [comment]
Thanks for the review, updated all said (apologies for the delay).

**Review by dipikabh — APPROVED** [review]

**Comment by dipikabh** [comment]
Thanks a lot @CanadaHonk for the updates! Much appreciate your work on this feature! 🙏

**Pull request merged**